apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.scss

```diff
@@ -85,3 +40,81 @@
   }
 }
 
+.feedback-container {
+  @include flex-center;
+  flex-direction: column;
+  background-color: transparent;
+
+  .feedback-form {
+    background-color: transparent;
+
+    @include flex-center(space-between, center);
+    flex-direction: column;
+  }
+}
+
+.copyright {
+  margin-top: 20px;
+  font-size: 12px;
+  color: #777;
+}
+
+::ng-deep {
```

Comment:
What is this big override about?

Reply:
It is to allow me target the mdc variables of the angular material component to apply styles that would achieve the desired outcome

Reply:
I don't like it. That is reason you customized it in the first place